### PR TITLE
style: 💄 hints for translators

### PR DESCRIPTION
**Pull request by MarcoEidinger**
Some of the related UIs

Client_CardsForScene

Client_Card_Edit

Client_AnchorImage

Client_AnchorImage_Edit

### CODE REVIEW

**Comment by MarcoEidinger** [comment]
@pgobriensap before requesting translations I added some hints to give them a better context of the vocabulary and terms as well as the use case. Please review

**Comment by MarcoEidinger** [comment]
@pgobriensap also were the texts verified by UX KM? Some of the terms used, like `Delete` (I would expect `Remove`) or `Save` (I would expect `Done` as data are not immediately saved and Done is used in another form), seem odd to me

**Comment by MarcoEidinger** [comment]
> @MarcoEidinger I used what is on the files in the [Final MVP Figma](https://www.figma.com/file/kFQGKYE7hnN8eL4jt1vzAH/AR-Creation?node-id=879%3A22157). I can reach out to recommend Remove and Done as it makes more sense for the context.

If you could double-check with them if those texts were vetted by KM and provide suggestion that would be great. Otherwise we could ask Andy for input on the texts

File: Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,64 +1,124 @@
...
-"Dimensions" = "Dimensions"
+/* XFLD: title of section in which user specifies physical dimensions (width) of image anchor */
+"Dimension" = "Dimension"
+/* XFLD: explanation text what kind of width is needed  */
 "Enter the width of the real-world image." = "Enter the width of the real-world image.";
+/* XFLD: label of input field for image anchors physical width  */
 "Width" = "Width";
+/* XMSG: instruction text on how to choose an image anchor  */
 "Tap the area below to upload the anchor image." = "Tap the area below to upload the anchor image.";
+/* XBUT: to remove an image anchor   */
```
Comment (pgobriensap) [inline Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings:70]: Delete can be changed to Remove.

File: Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,64 +1,124 @@
...
 "Custom Cover Image" = "Custom Cover Image";
+/* XFLD: placeholder text used in "Content" input field */
 "URL" = "URL";
+/* XFLD: placeholder text used in "Action Button" input field */
 "Label" = "Label";
+/* XBUT: button text used in form in which user edits card details of given AR annotation */
 "Update" = "Update";
+/* XBUT: button text used in form in which user creates new card (details) of new AR annotation */
 "Create" = "Create";
+/* XBUT: button text used in form in which user creates new card (details) of new AR annotation */
```
Comment (pgobriensap) [inline Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings:47]: Save can be changed to Done.

**Review by pgobriensap — CHANGES_REQUESTED** [review]
I reached out to Sang and she said these changes are okay. 

Done is already a part of the file for the marker flow. I'm not sure how different hints can apply to both scenarios as the string will only map to one value.

File: Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings (hunk 2 above)
Comment (MarcoEidinger) [reply Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings]: Done

File: Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings (hunk 1 above)
Comment (MarcoEidinger) [reply Sources/FioriAR/ARCards/Resources/en.lproj/Localizable.strings:68]: Done

**Review by pgobriensap — APPROVED** [review]

**Pull request merged**